packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1535,12 +1535,18 @@ export class GarbageCollector implements IGarbageCollector {
             // Events generated:
             // InactiveObject_Loaded, SweepReadyObject_Loaded
             if (usageType === "Loaded") {
-                this.mc.logger.sendErrorEvent({
```

Comment:
would it make sense to make this configurable? so that as the error trend down, it could be converted? i don't know if you'd also what the differentiation you've added on state

Reply:
I'm not completely sure what you mean, do you mean something like feature flags, or something else altogether?